solc/CommandLineInterface.cpp

```diff
@@ -918,12 +917,64 @@ void CommandLineInterface::serveLSP()
 		solThrow(CommandLineExecutionError, "LSP terminated abnormally.");
 }
 
-static bool isBytecodeChar(char c)
+static bool isBytecodeString(string s)
 {
-	return isxdigit(c) || c == '_' || c == '$';
```

Comment:
Unrelated to this PR, but:
> isxdigit(std::locale) | checks if a character is classified as a hexadecimal digit by a locale(function template)

Perhaps we'd be better off with just a manual check and could make it `constexpr` that case.